tee-worker/core-primitives/substrate-sgx/externalities/src/lib.rs

```diff
@@ -110,6 +110,12 @@ pub trait SgxExternalitiesTrait {
 	/// Get the next key in state after the given one (excluded) in lexicographic order.
 	fn next_storage_key(&self, key: &[u8]) -> Option<Vec<u8>>;
 
+	/// Iter prefix
```

Comment:
Consider changing doc to `Reads all values under given prefix`

Reply:
Done ✅